packages/flutter_tools/lib/src/commands/build_web.dart

```diff
@@ -186,3 +186,5 @@ class BuildWebCommand extends BuildSubCommand {
     return FlutterCommandResult.success();
   }
 }
+
+const String _wasmFlag = 'wasm';
```

Comment:
Can you actually add this here: https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/runner/flutter_command.dart#L100